static/panes/compiler.ts

```diff
@@ -3329,8 +3333,8 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         const cacheName = `asm/${instructionSet}/${opcode}`;
         const cached = OpcodeCache.get(cacheName);
         if (cached) {
-            if (cached.found) return cached.data;
-            throw new Error(cached.data);
+            if (cached.found) return cached.data as AssemblyInstructionInfo;
```

Comment:
@jeremy-rifkin this is the bit I think I could do better on. Actually we could change the design here - what would you suggest? Maybe get rid of `found` and use the presence or absence of `data` (with a separate `errorString`?)

Reply:
Good spot, I think that sounds perfect to me!